sphinx_ncs_theme/layout.html

```diff
@@ -14,6 +14,17 @@
   href="{{ pathto('_static/images/favicon.ico', 1) }}"
 />
 {% endblock %} {%- block extrabody %}
+
+<!-- Google Tag Manager (noscript) -->
+<noscript>
+  <iframe src=https://www.googletagmanager.com/ns.html?id=GTM-WF4CVFX
+  height="0"
+  width="0"
+  style="display:none;visibility:hidden">
+  </iframe>
+</noscript>
+<!-- End Google Tag Manager (noscript) -->
```

Comment:
we should avoid hardcoding this in the template, but:

1. make it optional (using a boolean varible, off by default)
2. ID must be a parameter

Reply:
Thank you for the input! I assume I make it optional with 
`{%- if not somevalue|tobool %}`
but where do I set this value? Similarly, if the id is set with 
`<iframe src=https://www.googletagmanager.com/ns.html?id={{some_value}}`
where is this value taken from?

Alternatively, is there a way to add this piece of code to the html body without altering the theme?